.github/workflows/test.yml

```diff
@@ -54,14 +54,38 @@ jobs:
         -   name: Install package with test dependencies
             run: |
                 pip install dist/*.whl
-                pip install '.[test,scanorama]'
+                pip install '.[test]'
 
         -   name: Test with pytest
             run: |
-                if [ ${{ matrix.os }} = "macos-latest" ]; then
-                    IGNORE="--ignore=tests/integration/"
-                fi
-                python -m pytest --cov --cov-append --cov-report=term-missing $IGNORE -vv
+                python -m pytest --cov --cov-append --cov-report=term-missing --ignore=tests/integration/ -vv
+        -   name: Upload coverage
+            env:
+                CODECOV_NAME: ${{ matrix.os }}-${{ matrix.python }}
+            run: |
+                codecov --no-color --required --flags unittest
+
+    integration:
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                python: [3.7, 3.9]
+                os: [ubuntu-latest]
+        steps:
+        -   uses: actions/checkout@v2
+        -   name: Set up Python ${{ matrix.python }}
+            uses: actions/setup-python@v2
+            with:
+                python-version: ${{ matrix.python }}
+
+        -   name: Install dependencies
+            run: |
+                python -m pip install --upgrade pip
+                pip install '.[test,scanorama,scvi]'
+
+        -   name: Test with pytest
+            run: |
+                python -m pytest --cov --cov-append --cov-report=term-missing -k integration -vv
```

Comment:
Separate GA integration test ;)

Reply:
ah, yes... for some reason i only read this as code coverage ^^. but the `integration` parameter in there is key ;)